frontend/package.json

```diff
@@ -14,8 +14,9 @@
     "@mapbox/mapbox-gl-geocoder": "^5.0.1",
     "@mapbox/mapbox-gl-language": "^0.10.1",
     "@placemarkio/geo-viewport": "^1.0.1",
-    "@sentry/react": "^7.56.0",
-    "@sentry/tracing": "^7.56.0",
+    "@sentry/react": "^7.60.1",
+    "@sentry/tracing": "^7.60.1",
+    "@sentry/browser": "^7.60.1",
```

Comment:
I couldn't find any instances where `@sentry/browser` package is being utilized. From the React | Sentry Docs 

> All methods available in `@sentry/browser` can be imported from @sentry/react.

Since we are already using `@sentry/react` in our project, it might be a good idea to consider removing the unused `@sentry/browser`.